exercises/parallel-letter-frequency/test/Tests.hs

```diff
@@ -80,3 +86,25 @@ specs = do
       it "all three anthems, together, 1 worker" $ testAllAnthems 1
 
       it "all three anthems, together, 4 workers" $ testAllAnthems 4
+
+      let moreAnthems = take 100 $ cycle anthems
+
+      let runBench n = benchmarkWith' (CM.defaultConfig { timeLimit = 0.1 }) (makeBench moreAnthems n)
+
+      it "multiple workers run faster than 1" $ do
+        setNumCapabilities 4
+
+        putStrLn "Run with a single worker"
+        singleWorkerMetric <- measuredMetric <$> runBench 1
+
+        let workers = 4
+        putStrLn $ "Run with " ++ show workers ++ " workers"
+        multiWorkerMetric <- measuredMetric <$> runBench workers
+
+        (singleWorkerMetric / multiWorkerMetric) `shouldSatisfy` (> 1.1)
+
+measuredMetric :: Report -> Double
+measuredMetric = B.estPoint . anMean . reportAnalysis
+
+makeBench :: [Text] -> Int -> Benchmarkable
+makeBench texts n = nf (frequency n) texts -- am I right that I should use nf and not whnf?
```

Comment:
`nf` seems preferable, but it accepts a smaller set of types than `whnf`.

In this case, because the returned value is a lazy `Map`, I guess that using `whfn` could give artificially lower and incorrect benchmark values.

I didn't tested though.